application/src/test/resources/org/opentripplanner/apis/gtfs/queries/rental-vehicle.graphql

```diff
@@ -19,5 +19,7 @@
       networkId
       url
     }
+    currentFuelPercent
+    currentRangeMeters
```

Comment:
Lets discuss if using something like

```
currentEnergy {
  fuel
  range
}
``` 

is good design. I honestly don't know.

Reply:
Yeah, It's in line with other properties in the schema. I don't know if I really like energy in this case, maybe autonomy is more suited? I'm not sure because I'm not an english native speaker.

Reply:
Autonomy is less understandable for me than currentEnergy. What about
```
fuel {
   percent
   range
}
```
I'm not sure if that is any better.

Reply:
Could be `currentFuel` also, but I'm not sure if that is necessary.

Reply:
I think I agree.